### DISCUSSION TITLE
How to use locator inside a Page Object

### DISCUSSION THREAD

**Discussion by incoming-th** [codeceptjs/CodeceptJS#4251] [Q&A] [answered]
Hi,

Everytime I try to use locator() in Page Object I got an error:

**locator is not defined**

![image](https://github.com/codeceptjs/CodeceptJS/assets/26375917/498def52-00a8-4d21-ad3f-020952007abd)

I believe it needs to be imported, but how? I cannot find this in the documentation.

Thanks

**Comment by kobenguyent** [comment, marked as answer]
I think it should be locate() instead of locator()

**Reply by incoming-th** [reply]
@kobenguyent You are absolutely right, I may need a break after 15 hours of work and fix my dyslexia.
